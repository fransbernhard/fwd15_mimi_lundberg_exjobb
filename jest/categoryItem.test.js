import React, { Component } from 'react';
import renderer from 'react-test-renderer';
import CategoryItem from '../src/components/categoryItem';

describe('components', () => {
  describe('<CategoryItem/>', () => {
    it('renders correctly', () => {
      const tree = renderer.create(
        <CategoryItem
          onClick={jest.fn}
          category="paint"
        />
      ).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
 
