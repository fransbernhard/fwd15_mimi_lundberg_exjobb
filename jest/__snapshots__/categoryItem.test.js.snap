// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <CategoryItem/> renders correctly 1`] = `
<div>
  <li
    className="category"
    onClick={undefined}
  >
    paint
  </li>
</div>
`;
